src/components/PeopleTablet/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,63 @@
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../../types';
+
+interface Props {
+  person: Person;
+}
+export enum Sex {
+  MALE = 'm',
+  FEMALE = 'f',
+}
+
+export const PersonLink: React.FC<Props> = (props) => {
```

Comment:
```suggestion
export const Person: React.FC<Props> = (props) => {
```
Just person.
The person link is just a link component